README.md

```diff
@@ -15,5 +15,12 @@ where year can just in 2016 - 2020
 ## Build Testing Server
 Here is the reference I made. [Go!!](https://mozixreality.github.io/Blog/featured/D20210503)
 
+## Build Local Testing Server
+
+```
+docker-compose up --build -d 
```

Comment:
I believe `docker-compose.yml` is used for local testing? or since the `docker-compose-dev.yml` is almost the same as `docker-compose.yml`, we can just use `python -m http.server --cgi 8888` for testing.